MachineLearningPrototype/utils/preprocess_data.py

```diff
@@ -66,8 +66,10 @@ def extract_jenkins_link_and_testname(content):
     test_names = []
 
     pattern = r"(https:\/\/.+\/job\/Test_openjdk\d+.+\/\d+)"
-    adoptium_jenkins_links = re.findall(pattern, content)
-    jenkins_links += adoptium_jenkins_links
+    splitted_content = content.split()
```

Comment:
Since `splitted_content` has been added here, the one in pattern3 should be removed.